src/markdown-pages/opentelemetry-masterclass/fundamentals/index.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Chapter one: OpenTelemetry fundamentals'
+template: 'LabOverviewTemplate'
+description: 'Chapter one: OpenTelemetry fundamentals'
+procIdx: 1
+---
+
+<Intro>
+
+In this first chapter, you learn the fundamentals of OpenTelemetry. Specifically, you’re introduced to core concepts and components of the OpenTelemetry standard and implementation. This will help you learn learn how, when, and why you use it in your applications.
+
+</Intro>
+
+<FlexWrapper>
+
+<Objectives>
+
+## Objectives
```

Comment:
Do you use anchors for headers? Goes for all doc!

Reply:
Could you clarify what you mean here? You can [link](https://build-56e2f14b-7fbf-43be-9790-b3778d364a58.gtsb.io/opentelemetry-masterclass/fundamentals#objectives) directly to any of the headers.

Reply:
I meant the headers in a doc - we use ## Objectives [#objectives]

Reply:
Interesting. No, the dev site automatically renders anchors with headers.